### ISSUE TITLE
Enable training underground?

### ISSUE THREAD

**Issue by derSteFfi** [ja2-stracciatella/ja2-stracciatella#239]
No kind of training is allowed underground right now
- Militia
- Teacher/Student
- Practice

While Militia makes kind of sense, maybe we should allow the other two. Opinions?

**Comment by JAsmine-ja2** [comment]
Yeah, probably

- repair
- doctor
- training (stats)
- teacher/student

sould all be allowed underground.

**Comment by selaux** [comment]
But isn't this also disabled in vanilla? If so we would have to hide it behind some switch.

**Comment by lynxlynxlynx** [comment]
I'm not sure that makes sense for something this trivial. I'd skip that complexity.

**Comment by selaux** [comment]
I agree that this is not a big deal and i'm not opposed to the idea. I just think that this might influence the playstyle somehow, by making the underground a safe-haven to regroup. I didn't experience any underground attacks in my plays until now and, correct me if i'm wrong, I dont think the enemy attacks underground sectors. So a player having some mercs being attacked in a bad situation might just move them under ground and wait for them to heal or repair their stuff.

**Comment by lynxlynxlynx** [comment]
I think bugs can attack you underground in sci-fi mode. Nobody else moves there.
While it could be cheesed, it's hard to justify the prohibition.

**Comment by JAsmine-ja2** [comment]
Okay, my 50 cents on this:
In Vanilla, the player was unable to do stuff like repairing or training underground. I don't know if this was the way the devs really wanted it to be, but it was this way and I'm used to it (and don't mind it).
If we would now allow the player to do everything in underground sectors, it would probably not have much (any?) impact on the playstyle.
The real question is perhaps, how much we want to alter the game. How much vanilla we want to change or take away completely.
My opinion is: as little as possible.
Changing a little (trivial) thing here and another aspect there can quickly lead to a 1.13-like scenario (started out as a stable modding platform but drifted away from this. is now almost impossible to use as modding platform due to countless new features).
So, either we should leave things here as they are (player is unable to do anything underground) or we should change it and make it an optional thing in game.json
(player is able to give assignments to mercs who are underground = true/false)

**Comment by derSteFfi** [comment]
So, my humble opinion on this:
I, too, don't want to make a new 1.13. I do not want to add features, just fix bugs and externalize stuff and keep the codebase up to date. In this case, i do not oppose to a config switch enabling training (except militia) underground but do not need it. Since this makes sense (why not teach someone medical skills in a bunker?) I'd consider it not being possible in Vanilla a bug. So I am pro-fix :)

Of course, no milita shall be trained underground, no vehicle may be mounted, etc. For the other issue (repairing underground, #152 ) as mentioned there, I see it as a bug. Therefore I am pro-fixing this one. That's also why I opened this issue as a seperate one to discuss.

So, in conclusion:
doctor/patient is already allowed underground, repair should be in my opinion, training (practise and teacher/student) can be and should be configurable if implemented.

**Comment by lynxlynxlynx** [comment]
Repeating what I said above: I agree with you.

Can we have the general discussion in #207? It feels like I'm talking to myself.
